### PR TITLE
Adding nb_conda_store_kernels 0.1.3

### PR DESCRIPTION
**Pull request by costrouc**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by costrouc** [comment]
Ready for review and merge. Currently this package only runs on linux. Will have osx and windows in the next releases.

**Comment by costrouc** [comment]
@conda-forge/help-python This is ready for review. Thanks!

File: recipes/nb_conda_store_kernels/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - conda-store >=0.4.4
+    - jupyter_client
+    - traitlets
```
Comment (goanpeca) [inline recipes/nb_conda_store_kernels/meta.yaml:24]: Very minor nitpick @costrouc but it helps to parse the recipes visually and update them if the requirements are liste in alphabetical order :)

**Review by goanpeca — APPROVED** [review]

File: recipes/nb_conda_store_kernels/meta.yaml (hunk 1 above)
Comment (costrouc) [reply recipes/nb_conda_store_kernels/meta.yaml:24]: Great in the future I will alphabetically sort them. Thanks for merging!

**Pull request merged**